src/main/java/games/strategy/engine/lobby/server/db/Database.java

```diff
@@ -123,7 +123,7 @@ private static void ensureDbTablesAreCreated(final Connection conn) {
         if (!existing.contains("TA_USERS")) {
           final Statement s = conn.createStatement();
           s.execute("create table ta_users" + "(" + "userName varchar(40) NOT NULL PRIMARY KEY, "
-              + "password varchar(40) NOT NULL, " + "email varchar(40) NOT NULL, " + "joined timestamp NOT NULL, "
+              + "password varchar(60) NOT NULL, " + "email varchar(40) NOT NULL, " + "joined timestamp NOT NULL, "
```

Comment:
I had an existing Derby database when I ran my tests earlier, and creating a user failed because it couldn't store the new longer password in the existing field.  I guess I'm just concerned that if we are going to run Derby and Postgres in parallel for some period of time, what's the upgrade path for the existing Derby database in production?  Even if we're just going to run an `ALTER TABLE` manually, we need to keep track of that action item when the release containing this code is deployed to the lobby.